packages/ui/src/components/va-modal/plugin/index.ts

```diff
@@ -3,14 +3,15 @@ import { defineVuesticPlugin, defineGlobalProperty } from '../../../services/vue
 import { createModalInstance } from '../modal'
 import { ModalOptions } from '../types'
 
+let modalInstance: any
```

Comment:
```suggestion
```

This will cause sideeffect... Vuestic can be used in multiple vue apps at the same time, so different modals can be closed.